datasets/br/tcu_debarred/crawler.py

```diff
@@ -0,0 +1,90 @@
+from zavod import Context, helpers as h
+from rigour.ids.stdnum_ import CPF, CNPJ
+
+def crawl_item(input_dict: dict, context: Context):
+    """
+    Creates an entity from the raw data.
+
+    :param input_dict: Data dict extracted from source.
+                       Should have at least following keys:
+                        - cpf_cnpj
+                        - nome
+                        - processo
+                        - deliberacao
+                        - data_transito_julgado
+                        - data_final
+    :param context: The context object.
+    """
+
+    entity = context.make("LegalEntity")
+
+    raw_tax_number = input_dict["cpf_cnpj"]
+
+    # If it's length 18 with punctuation, then it is a CNPJ (XX.XXX.XXX/XXXX-XX)
+    if len(raw_tax_number) == 18:
+        entity.add_schema("Company")
+        tax_number = CNPJ.normalize(raw_tax_number)
+        prefix = "br-cnpj"
+
+    # If it's length 14 with punctuation, then it is a CPF (XXX.XXX.XXX-XX)
+    elif len(raw_tax_number) == 14:
+        entity.add_schema("Person")
+        tax_number = CPF.normalize(raw_tax_number)
+        prefix = "br-cpf"
```

Comment:
you'll need to handle normalize returning None here - perhaps you can use normalize instead of the length condition?

Reply:
oh sorry I see how it's used now